bundles/org.openhab.ui/web/src/pages/settings/persistence/strategy-picker.vue

```diff
@@ -37,29 +37,62 @@
 
 <script>
 import { f7 } from 'framework7-vue'
+import cloneDeep from 'lodash/cloneDeep'
 
 export default {
   props: {
     title: String,
     name: String,
     strategies: Array,
     value: Array,
+    suggested: Array,
     disabled: Boolean
   },
   emits: ['strategies-selected'],
   data () {
     return {
       smartSelectParams: {
         view: f7.view.main,
-        openIn: 'popup'
+        openIn: 'popup',
+        virtualList: true,
```

Comment:
Why use a virtual list? We don't need to render a huge amount of entries, so we shouldn't need a virtual list here.